Update Guest Avatar Images

armandohimself opened the pull request:
- Query will pull in a guest's avatar and if not a place holder will show a gray avatar profile image. 
- Styling broke creating an event and I had to create a hot fix for it.
- Default images are shown to the diner if a chef has not uploaded an image of themselves.

armandohimself committed:
Update Guest Avatar Images

- Query will pull in a guest's avatar and if not a place holder will show a gray avatar profile image. 
- Styling broke creating an event and I had to create a hot fix for it.

armandohimself committed:
Text Changes

Update how many reservations on chef side.

JoshuaArcyLHarrison reviewed (approved):
Looks good. The avatar profile pic is transparent which might cause problems in the future, but it's fine for now.

armandohimself committed:
Chef Profile Images

Updated the image to not be a transparent image.

armandohimself committed:
Formatting Commit

Curious to see if I'll have an easier time to see my changes if I commit the formatting first then my changes.

armandohimself committed:
Small Input Solution

- Got to this point and realized this was not a good idea with how the date is formatted. I'll move the date and time into their own lines.

armandohimself committed:
Moved Start & End Date & Time to One Line

I don't like it but for MVP this is okay.

armandohimself committed:
Added End Date to Event Creation

When a chef creates an event
They can set a start & end a date & time
So their guests understand event duration

armandohimself committed:
Viewing End Date on Event Details

You can view end date on any event created

armandohimself committed:
Accidental Commit

Fixed what was committed.
